tests/test_flujo_requisiciones.py

```
@@ -90,9 +94,10 @@ def test_creacion_requisicion_envia_correos(client, mocker):
 
     response = client.post('/requisiciones/crear', data=data, follow_redirects=True)
     assert response.status_code == 200
-    # correo al solicitante y a Almacén
-    assert enviar.call_count >= 1
-    print("Correos enviados (creación):", enviar.call_args_list)
+    assert enviar.call_count == 2
+    destinatarios_list = [call.args[0] for call in enviar.call_args_list]
```

Ruff (0.11.9)

Redefinition of unused `call` from line 4

(F811)